chat-plugins/tickets.js

```diff
@@ -0,0 +1,487 @@
+}
+
+function ticketForm(user) {
+	let buf = `>view-ticket-create\n|init|html\n|title|Help Ticket\n|pagehtml|<div class="pad"><h2>Request help from global staff</h2><b>Whats going on?</b><details style="margin: 3px"><summary>I want to report someone</summary>`;
+	buf += `<br /><b>What do you want to report someone for?</b><br /><details style="margin: 3px"><summary>Someone is harrasing me</summary>If someone is harrasing you, click the appropriate button below and a global staff member will take a look. Consider using <code>/ignore [username]</code> if its minor instead.<br /><br />If your reporting harrasment in a battle, please save a replay of the battle.<br /><button class="button" name="send" value="/ticket submit PM Harrasment">Report harrasment in a private message (PM)</button> <button class="button" name="send" value="/ticket submit Battle Harrasment">Report harrasment in a battle</button> <button class="button" name="send" value="/ticket submit Chatroom Harrasment">Report harrasment in a chatroom</button></details>`;
+	buf += `<details style="margin: 3px"><summary>Someone is being inappropriate</summary>If a user has posted inappropriate content, has an inappropriate name, or has inappropriate pokemon nicknames, click the appropriate button below and a global staff member will take a look.<br /><br /><button class="button" name="send" value="/ticket submit Inappropriate Content">Report inappropriate content</button> <button class="button" name="send" value="/ticket submit Inappropriate Nickname">Report an inappropriate username</button> <button class="button" name="send" value="/ticket submit Inappropriate Pokemon Nicknames">Report inappropriate pokemon nicknames</button></details>`;
+	buf += `<details style="margin: 3px"><summary>Someone is timerstalling</summary>If someone is timerstalling in your battle, and the battle has <b>not</b> ended, click the button below and a global staff member will take a look.<br /><br /><button class="button" name="send" value="/ticket submit Timerstalling">Report timerstalling</button></details>`;
+	buf += `<details style="margin: 3px"><summary>I want to report a staff member</summary>If you have a complaint against a room staff member, please PM a Room Owner (marked with a #) in the room.<br /><br />If you have a complaint against a global staff member or room owner, please click the appropriate button below. Alternatively, make a post in <a href="http://www.smogon.com/forums/threads/names-passwords-rooms-and-servers-contacting-upper-staff.3538721/#post-6300151">admin requests</a>.<br /><br /><button class="button" name="send" value="/ticket submit Room Owner Complaint">Report a Room Owner</button> <button class="button" name="send" value="/ticket submit Global Staff Complaint">Report a Global Staff Member</button></details>`;
+	buf += `</details><br /><details style="margin: 3px"><summary>I want to appeal a punishment</summary><br /><b>What would you like to appeal?</b><br />`;
+	if (user.locked) {
```

Comment:
What about a semilocked check? (@Zarel - is semilocked just ``user.semilocked``?)

Reply:
We should probably handle semilocks in some way.

Reply:
Probably just exactly the same way.

Reply:
did this.